modules/nw-operator-cr.adoc

```diff
@@ -14,11 +14,19 @@ ifeval::["{context}" == "cluster-network-operator"]
 :operator:
 endif::[]
 
+ifeval::["{context}" == "post-install-network-configuration"]
+:post-install-network-configuration:
+endif::[]
+
 [id="nw-operator-cr_{context}"]
 = Cluster Network Operator configuration
 
 The configuration for the cluster network is specified as part of the Cluster Network Operator (CNO) configuration and stored in a custom resource (CR) object that is named `cluster`. The CR specifies the parameters for the `Network` API in the `operator.openshift.io` API group.
 
+ifdef::post-install-network-configuration[]
+NOTE: After cluster installation, you cannot modify the configuration for the cluster network provider.
```

Comment:
Place the note in the [admonition format](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#admonitions)

Reply:
@ahardin-rh fixed!